dashboard/app/models/resource.rb

```diff
@@ -92,7 +92,7 @@ def summarize_for_lesson_plan
       id: id,
       key: key,
       name: get_localized_property(:name),
-      url: url,
+      url: get_localized_property(:url),
```

Comment:
Do you have to update the `url` in `summarize_for_lesson_edit` function?

Reply:
nope! That's used for the levelbuilder editing interface, which we explicitly want to only interact with English content